app/javascript/react/screens/App/Plan/PlanReducer.js

```diff
@@ -35,102 +36,88 @@ const excludeDownloadDoneTaskId = (allDownloadLogInProgressTaskIds, taskId) =>
 const includeDownloadInProgressTaskId = (allDownloadLogInProgressTaskIds, taskId) =>
   allDownloadLogInProgressTaskIds ? allDownloadLogInProgressTaskIds.concat(taskId) : [taskId];
 
-const _formatPlanRequestDetails = data => {
+const processVMTasks = vmTasks => {
   const tasks = [];
-  if (data.miq_request_tasks && data.miq_request_tasks.length) {
-    data.miq_request_tasks.forEach(task => {
-      const taskDetails = {
-        id: task.id,
-        message: task.message,
-        transformation_host_name: task.options && task.options.transformation_host_name,
-        delivered_on: new Date(task.options.delivered_on),
-        updated_on: new Date(task.updated_on),
-        completed: task.message === 'VM Transformations completed' || task.message === 'VM Transformations failed',
-        state: task.state,
-        status: task.status,
-        options: {}
-      };
-
-      taskDetails.options.progress = task.options.progress;
-      taskDetails.options.virtv2v_wrapper = task.options.virtv2v_wrapper;
-
-      if (!task.diskSpaceCompletedGb) {
-        taskDetails.diskSpaceCompletedGb = '0';
-      }
+  vmTasks.forEach(task => {
```

Comment:
any chance vmTasks could ever be null/undefined?

Reply:
No, not in PlanReducer - i.e., we are always expected to have tasks in the Plan detail screen.
(PlanReducer is the reducer for Plan detail screen)